README.md

```diff
@@ -76,14 +87,18 @@ pipeline step for mabl. This can be written by hand or created via the
 -   continueOnPlanFailure: Continue if your plan/tests fail
 -   environmentId: Selects the environment to run deployments against
     (one of environmentId or applicationId is required)
--   restApiKey: The apiKey of the desired deployment workspace -
+-   restApiKeyName: The name of the API key secret of the desired deployment workspace -
     Required  
-      
+
+*Note* that if you want to select specific plan labels, then use the 
+`labels: ['label1','label2',...,'labeln'].toSet()` syntax in your pipeline step. The Generate Pipeline
+Script page in Jenkins does not append `toSet()`. If you do not use labels, then set the value to null,
+for example: `labels: null`
 
 **Pipeline Step Setup**
 
 ``` syntaxhighlighter-pre
-mabl applicationId: 'APP-ID-a', continueOnMablError: true, continueOnPlanFailure: true, environmentId: 'ENV-ID-e', restApiKey: 'REST-API-KEY'
+mabl applicationId: 'APP-ID-a', continueOnMablError: true, continueOnPlanFailure: true, environmentId: 'ENV-ID-e', restApiKeyName: 'REST-API-KEY-NAME'
 ```
 
 OR something like this:
```

Comment:
This one is `scripted pipeline` (I think).

Reply:
It is indeed.